src/main/scala/org/apache/spark/sql/snowflake/SnowflakeSparkUtils.scala

```diff
@@ -0,0 +1,49 @@
+package org.apache.spark.sql.snowflake
+
+import org.apache.spark.sql.execution.datasources.{LogicalRelation, SaveIntoDataSourceCommand}
+import org.apache.spark.sql.execution.datasources.jdbc.{JDBCRelation, JdbcRelationProvider}
+
+object SnowflakeSparkUtils {
+
+  private def getClassName(obj: Any): String = {
+    val shortClassNames = Seq("net.snowflake.spark.snowflake.SnowflakeRelation",
+      "net.snowflake.spark.snowflake.DefaultSource",
+      "org.apache.spark.sql.execution.datasources.jdbc.JdbcRelationProvider",
+      "org.apache.spark.sql.execution.datasources.jdbc.JDBCRelation")
+    val className = obj.getClass.getName
```

Comment:
We mostly query the spark plan statistic with these 4 classes, so use short name for them.